src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -373,5 +379,21 @@ protected override void Serialize(BlobBuilder builder, SectionLocation location)
                 NativeResourceWriter.SerializeWin32Resources(builder, _resources, location.RelativeVirtualAddress);
             }
         }
+
+        private class ResourceSectionBuilderFromRaw : ResourceSectionBuilder
+        {
+            private readonly Stream _resources;
```

Comment:
May want to comment that it was a deliberate decision to not `Dispose` this value and that caller needs to be aware of that and make sure to close it.

Reply:
Updated the doc comment for win32Resources to clarify this.